### PR TITLE
Users attempting to load plugins that have cocoapod dependancies will fail

### PR DESCRIPTION
**Pull request by sitefinitysteve**
This setup instruction will help users who are trying to load plugins that require cocoapods.

There's already a bunch of posts out there related to the strange error

```
Creating shallow clone of spec repo `master` from `https://github.com/CocoaPods/Specs.git`
[!] Unable to add a source with url `https://github.com/CocoaPods/Specs.git` named `master`.
You can try adding it manually in `~/.cocoapods/repos` or via `pod repo add`.
Processing node_modules failed. Error:Error: Command sandbox-pod failed with exit code 1
```

https://groups.google.com/forum/#!msg/nativescript/BFayGzYA1xM/WPFmhxbxBgAJ

### CODE REVIEW

File: start/ns-setup-os-x.md [hunk 1]
```diff
@@ -51,6 +51,14 @@ For Android development
     ```Shell
     brew install homebrew/versions/node012
     ```
+1. Install and configure cocoapods
```
Comment (ikoevska) [inline start/ns-setup-os-x.md:54]: Install and configure CocoaPods.

File: start/ns-setup-os-x.md [hunk 2]
```diff
@@ -51,6 +51,14 @@ For Android development
     ```Shell
     brew install homebrew/versions/node012
     ```
+1. Install and configure cocoapods 
+
+    ```Shell
+    sudo gem install cocoapods
```
Comment (ikoevska) [inline start/ns-setup-os-x.md:57]: You can combine the two commands inside one block. For example:

```Shell
sudo gem install cocoapods
pod setup
```

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by ErjanGavalji** [comment]
Applied with commit https://github.com/NativeScript/docs/commit/c85e9635f227799bdc6366a9a669041f9ad24215

**Pull request closed without merging**